src/sql/src/session/vars.rs

```diff
@@ -2151,13 +2113,13 @@ impl SystemVars {
             || name == STORAGE_STATISTICS_COLLECTION_INTERVAL.name()
             || name == USER_STORAGE_MANAGED_COLLECTIONS_BATCH_DURATION.name()
             || is_upsert_rocksdb_config_var(name)
-            || self.is_persist_config_var(name)
+            || self.is_dyncfg_var(name)
             || is_tracing_var(name)
     }
 
     /// Returns whether the named variable is a persist configuration parameter.
-    fn is_persist_config_var(&self, name: &str) -> bool {
-        self.persist_configs.entries().any(|e| name == e.name())
+    fn is_dyncfg_var(&self, name: &str) -> bool {
+        self.dyncfgs.entries().any(|e| name == e.name())
```

Comment:
It seems this is linear for every `name`, so it could potentially be quadratic for all `is_***_config_var` calls. Just noting this, it doesn't seem worse than what we already have, but could be improved by using a map to lookup the value.

Reply:
Agreed